composer.json

```diff
@@ -23,6 +23,12 @@
         }
     },
     "scripts": {
+        "lint": [
+            "phpcs"
+        ],
+        "lint-fix": [
+            "phpcbf"
```

Comment:
Might as well run `phpcbf` and `phpcs` from within Composer, instead of running them like `./vendor/bin/phpcbf`